plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -217,6 +219,27 @@ static void log_muxer_params(struct ffmpeg_muxer *stream, const char *settings)
 	av_dict_free(&dict);
 }
 
+const char *get_stream_key(struct ffmpeg_muxer *stream)
+{
+	obs_service_t *service;
+	const char *stream_key;
+
+	service = obs_output_get_service(stream->output);
+	if (!service)
+		return NULL;
+	stream_key = obs_service_get_key(service);
+	return stream_key;
+}
+
+static void add_stream_key(struct dstr *cmd, struct ffmpeg_muxer *stream)
+{
+	const char *stream_key;
+
+	stream_key = get_stream_key(stream);
```

Comment:
I think it's not necessary to pass stream key in the command when it is not HLS. We can store stream_key as a member field in struct ffmpeg_muxer and set only in ffmpeg_hls_mux_start.